### ISSUE TITLE
feat: Default or already-received arguments for fragment prompts

### ISSUE THREAD

**Issue by TehBrian** [TehBrian/projutil-node#3]
`README.md` should be an optional fragment in `JavaPaperPlugin` much like `LICENSE` already is. However, to achieve this, `README.md` would require a prompt for the project name and project description. If the user traces the fragment `README.md` by itself, `README.md` should prompt for this information. However, `JavaPaperPlugin` already prompts the user for this info, and it would be unnecessary to ask the user twice. Somehow, `JavaPaperPlugin` should be able to feed already-asked questions into the `README.md` fragment. This may be able to be achieved by passing an object to `Fragment#prompt` with the already known information, and having the prompt question type be null if the information is already there; however, ideally, we would have more abstraction built up around this concept.

tl;dr blah blah blah make it so prompts don't appear twice

**Comment by TehBrian** [comment]
Moving issue to [the new projutil repository](https://github.com/TehBrian/projutil).
